plaso/storage/sqlite/sqlite_file.py

```diff
@@ -431,6 +431,78 @@ def _ReadAndCheckStorageMetadata(self, check_readable_only=False):
     self.serialization_format = metadata_values['serialization_format']
     self.storage_type = metadata_values['storage_type']
 
+  def _UpdateEventDataIdentifierAfterDeserialize(self, event):
+    """Updates the event data identifier of an event after deserialization.
+
+    Args:
+      event (EventObject):
+    """
+    row_identifier = getattr(event, 'event_data_row_identifier', None)
+    if row_identifier is not None:
+      event_data_identifier = identifiers.SQLTableIdentifier(
+          'event_data', row_identifier)
+      event.SetEventDataIdentifier(event_data_identifier)
+
+      delattr(event, 'event_data_row_identifier')
```

Comment:
Shouldn't this set to None, not delete the attribute? event_data_row_identifier part of the public api for the event container. This goes for the other methods too.

Reply:
I'm mimicking the current behavior with `del`. We can chat more about about these special attributes and how to best handle them in our conversation.